Packs/CrowdStrikeFalcon/Playbooks/playbook-Command_and_Scripting_Interpreter.yml

```diff
@@ -0,0 +1,1717 @@
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      key:
+        simple: CrowdStrike.Detection.grandparentprocess
+      value:
+        simple: ${incident.grandparentprocess}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1810,
+          "y": -890
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "97":
+    id: "97"
+    taskid: 7a918996-edd9-4e76-872e-18b104c0f7c2
+    type: regular
+    task:
+      id: 7a918996-edd9-4e76-872e-18b104c0f7c2
+      version: -1
+      name: Set the parent Process Details
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Automations
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      append:
+        simple: "true"
+      key:
+        simple: CrowdStrike.Detection
+      value:
+        complex:
+          root: incident
+          accessor: parentprocess
+          transformers:
+          - operator: join
+            args:
+              separator:
+                value:
+                  simple: ','
+          - operator: Stringify
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: '{"parentprocess":'
+              suffix:
+                value:
+                  simple: '}'
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1410,
+          "y": -890
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "98":
+    id: "98"
+    taskid: c165d836-695b-4a31-8fe0-3d5fac6bef82
+    type: regular
+    task:
+      id: c165d836-695b-4a31-8fe0-3d5fac6bef82
+      version: -1
+      name: Set the Policy Actions
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Automations
```

Comment:
Do you need to provide a link for XSOAR 6.x, XSIAM, XSOAR 8 Cloud and XSOAR on-prem?
Btw, XSOAR 6.10 is a very old version.
For Cortex XSOAR 6.13, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.13/Cortex-XSOAR-Administrator-Guide/Create-Custom-Filter-and-Transformer-Operators
For Cortex XSOAR 8 Cloud, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Scripts
For Cortex XSOAR on-prem, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Scripts

Reply:
For now, xsoar 6 and 8 
later on XSIAM.